internal/nvmeof/nvmeof.go

```diff
@@ -342,6 +342,12 @@ func (gw *GatewayRpcClient) CreateListener(ctx context.Context, subsystemNQN str
 
 		return nil // Listener already created, no error
 	}
+	if resp.GetStatus() == int32(syscall.EREMOTE) { // Handle the stashed listener case
```

Comment:
There are now 3 `if resp.GetStatus()` checks. It is cleaner to make it a switch/case statement.